### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *fassanity* failed. **[Why did it fail? →](https://app.netlify.com/projects/fassanity/deploys/6900731d24623f0008df807e?deploy_diagnostics=true)**

### ISSUE THREAD

**Issue by amingione** [amingione/fas-sanity#115]
### <span aria-hidden="true">❌</span> Deploy Preview for *fassanity* failed. **[Why did it fail? →](https://app.netlify.com/projects/fassanity/deploys/6900731d24623f0008df807e?deploy_diagnostics=true)**

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 516ff3987d47835a5dae25635ec116228ee0c04c |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/fassanity/deploys/6900731d24623f0008df807e |

_Originally posted by @netlify[bot] in https://github.com/amingione/fas-sanity/issues/114#issuecomment-3454981544_
